src/__tests__/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
- [ ] Kindly write more unit tests for the other components using React Testing Library and Jest.